sources/packages/web/src/views/aest/student/applicationDetails/ApplicationRestrictionsManagement.vue

```diff
@@ -37,12 +44,23 @@ export default defineComponent({
       required: true,
     },
   },
-  setup() {
+  setup(props) {
     const { emptyStringFiller } = useFormatters();
+    const currentApplicationId = ref<number>(0);
```

Comment:
The application id ref w/o 0 initialization is working for me.
![image](https://github.com/user-attachments/assets/ee39495b-34f9-4142-9e2d-02ffb6cbe8de)